crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -129,6 +129,16 @@ def finished?
       current_step == upgrade_steps_6_7.last
     end
 
+    def cancel_allowed?
+      [
+        :upgrade_prechecks,
+        :upgrade_prepare,
+        :admin_backup,
+        :admin_repo_checks,
+        :admin_upgrade
```

Comment:
i have solved it now by adding an additional check if the `admin_upgrade` is running